org.osgi.test.junit5.cm/src/main/java/org/osgi/test/junit5/cm/BlockingConfigurationHandlerImpl.java

```diff
@@ -41,6 +41,7 @@ public boolean update(Configuration configuration, Dictionary<String, Object> di
 		CountDownLatch latch = createCountdownLatchUpdate(configuration.getPid());
 		boolean updatedBecauseDifferent = configuration.updateIfDifferent(dictionary);
 		if (!updatedBecauseDifferent) {
+			updateMap.remove(configuration.getPid());
```

Comment:
These fixes look like a separate bug which should be done separately.

Reply:
yep.
thats why its an deparate commit.

but can male a single pr of this